public/index.html

```diff
@@ -570,146 +574,156 @@
-        </div>
 
-        <hr/>
+          <hr/>
 
-        <h2 class="i18n" data-i18n="ftm-what-to-do-header">How to Donate PPE</h2>
-        <p>Our network of volunteers has created a global map of over 1700+ locations, like hospitals and clinics, where you can donate your masks, gowns, and other PPE.</p>
-        <ol>
-          <li class="i18n" data-i18n="[html]ftm-what-to-do-1"><b>Use our map</b> below to find a hospital or health center near you accepting donations.</li>
-          <li class="i18n" data-i18n="[html]ftm-what-to-do-2"><b>Follow their instructions</b> for what they accept and how to drop it off.</li>
-          <li class="i18n" data-i18n="[html]ftm-what-to-do-3"><b>If you are donating opened packages</b>, please put them in a ziploc bag with a note saying whether they have been used, whether anyone in your household has been sick or travelled outside the country, and the expiration date if known.</li>
-        </ol>
+          <h2 class="i18n" data-i18n="ftm-what-to-do-header">How to Donate PPE</h2>
+          <p>Our network of volunteers has created a global map of over 1700+ locations, like hospitals and clinics, where you can donate your masks, gowns, and other PPE.</p>
+          <ol>
+            <li class="i18n" data-i18n="[html]ftm-what-to-do-1"><b>Use our map</b> below to find a hospital or health center near you accepting donations.</li>
+            <li class="i18n" data-i18n="[html]ftm-what-to-do-2"><b>Follow their instructions</b> for what they accept and how to drop it off.</li>
+            <li class="i18n" data-i18n="[html]ftm-what-to-do-3"><b>If you are donating opened packages</b>, please put them in a ziploc bag with a note saying whether they have been used, whether anyone in your household has been sick or travelled outside the country, and the expiration date if known.</li>
+          </ol>
 
-        <div class="question i18n" data-i18n="[html]ftm-large-donation-question">Have <em>a lot</em> (500+ items) to donate?</div>
-        <p class="i18n" data-i18n="[html]ftm-large-donation-answer">We encourage large donors to use another site we work with to help route donations. Register at <a target="_blank" rel="noreferrer noopener" href="https://getusppe.org/give">getusppe.org/give</a>.</p>
+          <div class="question i18n" data-i18n="[html]ftm-large-donation-question">Have <em>a lot</em> (500+ items) to donate?</div>
+          <p class="i18n" data-i18n="[html]ftm-large-donation-answer">We encourage large donors to use another site we work with to help route donations. Register at <a target="_blank" rel="noreferrer noopener" href="https://getusppe.org/give">getusppe.org/give</a>.</p>
 
-        <div class="question">Can't find a donation site near you?</div>
-        <p>Visit <a target="_blank" rel="noreferrer noopener" href="https://getusppe.org/give">getusppe.org</a> to let them know what you have available.</p>
+          <div class="question">Can't find a donation site near you?</div>
```

Comment:
can you add i18n here?

![image](https://user-images.githubusercontent.com/5788989/77830810-80ead100-7101-11ea-8f1f-461b8d37ef9e.png)

Reply:
Yep!